version.txt

```diff
@@ -1 +1 @@
-0.51.0
```

Comment:
Don't update this one. Release-please will do it automatically.

Reply:
Ok, thanks.